skit.API/Controllers/Areas/CompanyOwner/C_CompaniesController.cs

```diff
@@ -10,7 +10,7 @@ namespace skit.API.Controllers.Areas.Companies;
 
```

Comment:
Namespace do zmiany, jest taki skrot ctrl+shift+R i tam zaznaczasz adjust namespace i ci sprawdza czy masz prawidlowe

Reply:
to poprawię ale już na nowym branchu bo mnie pojebie żeby znowu czekać na 2 approve